python-sdk/src/langwatch/client.py

```
@@ -713,11 +715,21 @@ def __set_langwatch_exporter(self, provider: TracerProvider) -> None:
             )
 
-        otlp_exporter = OTLPSpanExporter(
-            endpoint=f"{Client._endpoint_url}/api/otel/v1/traces",
-            headers=headers,
-            timeout=int(os.getenv("OTEL_EXPORTER_OTLP_TRACES_TIMEOUT", 30)),
-        )
+        ssl_verify_value = Client._get_ssl_config()
+        exporter_kwargs = {
+            "endpoint": f"{Client._endpoint_url}/api/otel/v1/traces",
+            "headers": headers,
+            "timeout": int(os.getenv("OTEL_EXPORTER_OTLP_TRACES_TIMEOUT", 30)),
```

Ruff (0.15.20)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)